### DISCUSSION TITLE
Inserting School Logo in Open Web UI

### DISCUSSION THREAD

**Discussion by tbendien** [open-webui/open-webui#869] [Q&A]
Hello I am about to start a pilot program at a high school to bring private AI to  small test group of teachers and students using OWUI and Ollama.

We would like to insert the school logo into the UI so that everyone can see that this is an official private AI service for the school.

It would be great if we can put the school logo where the OWUI logo appears now in the top left and have "powered by Open Web UI" shown there as well.

Would that be possible to implement? @tjbck

**Comment by tjbck** [comment]
https://docs.openwebui.com/enterprise/
